Android/BetterPickers/External-Dependency-Info.txt

```diff
@@ -0,0 +1,52 @@
+THIRD-PARTY SOFTWARE NOTICES AND INFORMATION
+Do not translate or localize
+
+Xamarin Components for BetterPickers incorporates 
+third party material from the projects listed below. The original copyright 
+notice and the license under which Microsoft received such third party 
+material are set forth below.  Microsoft reserves all other rights not 
+expressly granted, whether by implication, estoppel or otherwise.
+
+########################################
+# android-betterpickers
+# https://github.com/code-troopers/android-betterpickers
+########################################
+
+Copyright 2013 Derek Brameyer, Code-Troopers
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+########################################
+# END: android-betterpickers
+########################################
+
+########################################
+# android-switch-backport
+# https://github.com/BoD/android-switch-backport
+########################################
+
+Licensed under the Apache License, Version 2.0 (the "License"); 
```

Comment:
replace this text with the contents of this file...  

https://github.com/BoD/android-switch-backport/blob/master/NOTICE.txt